symphony-bdk-core/src/main/java/com/symphony/bdk/core/service/stream/StreamService.java

```diff
@@ -0,0 +1,288 @@
+    } catch (ApiException apiException) {
+      throw new ApiRuntimeException(apiException);
+    }
+  }
+
+  /**
+   * Get information about a particular stream.
+   *
+   * @param streamId The stream id
+   * @return The information about the stream with the given id.
+   * @see <a href="https://developers.symphony.com/restapi/reference#stream-info-v2">Stream Info V2</a>
+   */
+  public V2StreamAttributes getStreamInfo(String streamId) {
+    try {
+      return streamsApi.v2StreamsSidInfoGet(streamId, authSession.getSessionToken());
+    } catch (ApiException apiException) {
+      throw new ApiRuntimeException(apiException);
+    }
+  }
+
+  /**
+   * List attachments in a particular stream.
+   *
+   * @param streamId The stream id
+   * @param sinceInMillis Timestamp in millis of first required attachment
+   * @param toInMillis Timestamp in millis of last required attachment
+   * @param sort Attachment date sort direction : ASC or DESC (default to ASC)
+   * @return List of attachments in the stream with the given stream id.
+   * @see <a href="https://developers.symphony.com/restapi/reference#list-attachments">List Attachments</a>
+   */
+  public List<StreamAttachmentItem> getAttachmentsOfStream(String streamId, Long sinceInMillis, Long toInMillis, AttachmentSort sort) {
+    try {
+      return streamsApi.v1StreamsSidAttachmentsGet(streamId, authSession.getSessionToken(), sinceInMillis, toInMillis, null, sort.name());
+    } catch (ApiException apiException) {
+      throw new ApiRuntimeException(apiException);
+    }
+  }
+
+  /**
+   * Create a new single or multi party instant message conversation.
+   * At least two user IDs must be provided or an error response will be sent.
+   *
+   * The caller is not included in the members of the created chat.
+   *
+   * Duplicate users will be included in the membership of the chat but the
+   * duplication will be silently ignored.
+   *
+   * If there is an existing IM conversation with the same set of participants then
+   * the id of that existing stream will be returned.
+   *
+   * @param uids List of user IDs of participants. At least two user IDs must be provided
+   * @return The created IM or MIM
+   * @see <a href="https://developers.symphony.com/restapi/reference#create-im-or-mim-admin">Create IM or MIM Non-inclusive</a>
+   */
+  public Stream createInstantMessageAdmin(List<Long> uids) {
+    try {
+      return streamsApi.v1AdminImCreatePost(authSession.getSessionToken(), uids);
+    } catch (ApiException apiException) {
+      throw new ApiRuntimeException(apiException);
+    }
+  }
+
+  /**
+   * Deactivate or reactivate a chatroom via AC Portal.
+   *
+   * @param streamId The stream id
+   * @param active Deactivate or activate
+   * @return The information of the room after being deactivated or reactivated.
+   */
+  public RoomDetail setRoomActiveAdmin(String streamId, Boolean active) {
```

Comment:
Seems not to be documented in https://developers.symphony.com/restapi/reference

Reply:
whats the difference with the other method setRoomActive?